spec/enhanced_spec.sh

```
@@ -0,0 +1,219 @@
+  Describe 'Export Functions'
+    It 'exports package list'
+      mkdir -p "$AWESOME_DIR/test-pkg/.git"
+      touch "$AWESOME_DIR/test-pkg/test-pkg"
+      ln -s "$AWESOME_DIR/test-pkg/test-pkg" "$BIN_DIR/test-pkg"
+      
+      cd "$AWESOME_DIR/test-pkg" || exit
+      git init -q
+      git config user.email "test@test.com"
+      git config user.name "Test"
+      git remote add origin "https://github.com/test/test-pkg"
+      cd - > /dev/null
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)